rocAL/source/augmentations/geometry_augmentations/node_crop_resize.cpp

```diff
@@ -81,15 +93,31 @@ void CropResizeNode::update_node() {
 }
 
 void CropResizeNode::init(float area, float aspect_ratio, float x_center_drift, float y_center_drift) {
```

Comment:
Why interpolation type is not added?

Reply:
Previously CropResize node used vxExtRppResizeCrop which only supports bilinear interpolation. So the init functions didn't have the interpolation type arg. In this PR, we only add the CropResizeFixed support so I didn't change the existing CropResize API. Let me know if it should be changed as well

Reply:
Please make that change too.

Reply:
As discussed offline, will not be changing the existing APIs and the init functions in this PR